test/multiupload.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const test = require('tape');
+const Storj = require('../lib/index.js');
+var Readable = require('stream').Readable;
+var KeyPair = require('storj-lib/lib/crypto-tools/keypair');
+
+test.skip('Storj.js concurrent upload', function(done) {
```

Comment:
File uploaded to share progress.

This test is currently skipped and 1/2 implemented. When complete it will test for race conditions with concurrent uploads.